ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/event-handlers.md

```diff
@@ -42,12 +42,18 @@ interface Management1
 
 | Handler | Action Type | Action | Trigger |
 | ------- | ----------- | ------ | ------- |
+| CONFIG_VERSIONING | bash | FN=/mnt/flash/startup-config; LFN="`ls -1 $FN.*-* | tail -n 1`"; if [ -z "$LFN" -o -n "`diff -I 'last modified' $FN $LFN`" ]; then cp $FN $FN.`date +%Y%m%d-%H%M%S`; ls -1r $FN.*-* | tail -n +11 | xargs -I % rm %; fi | on-startup-config |
```

Comment:
please update the documentation template to enclose the action in backticks so we see it as code

Reply:
I put it as code block now because backtick was not working properly as in the code line are all sort of special characters including backticks. Hope that is ok this way?

Reply:
Remember to rerun molecule after this change. The CI is failing because the output is not matching.

Reply:
Ah just noticed, you changed the markdown directly, which will be overwritten every time we run the tests. You should change the template and regenerate all molecule where this is used. (cd molecule, make refresh-facts)

Reply:
oh, sorry. Did the changes now in the correct file and did regenerate all molecule